Configure travis-ci for portal

mdelapenya opened the pull request.

mdelapenya committed:
LPS-XXX Initialize Travis-ci config file

mdelapenya committed:
LPS-XXX Build using Oracle JDK and OpenJDK, versions 7 and 8

mdelapenya committed:
LPS-XXX Define Travis-ci lifecycle, skipping each stage

mdelapenya committed:
LPS-XXX Install tomcat as on dependency resolution time

mdelapenya committed:
LPS-XXX Set ANT_OPTS as env variable

liferay-continuous-integration commented:
<h1>All tests PASSED!</h1><p>Build Time: 47 minutes 30 seconds</p><h1>See the following links for more information.</h1><ul><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest(master)/679/">https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest(master)/679/</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-start(master)/679/">https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-start(master)/679/</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/672/">https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/672/</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-plugins(master)/558/">https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-plugins(master)/558/</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-backend(master)/558/">https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-backend(master)/558/</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-stop(master)/672/">https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-stop(master)/672/</a></li></ul>

mdelapenya committed:
LPS-XXX Execute tests

mdelapenya committed:
LPS-XXX FIrst execute ant all

mdelapenya committed:
LPS-XXX Run tests against MySQL

mdelapenya committed:
LPS-XXX Remove openJDK, as it is failing with this stack trace:

======= Backtrace: =========
/lib/x86_64-linux-gnu/libc.so.6(__fortify_fail+0x37)[0x7ffe3655ddb7]
/lib/x86_64-linux-gnu/libc.so.6(+0x109cb0)[0x7ffe3655ccb0]
/usr/lib/jvm/java-7-openjdk-amd64/jre/lib/amd64/libnet.so(Java_java_net_Inet4AddressImpl_getLocalHostName+0x190)[0x7ffdfb84f4a0]
[0x7ffe2c3a2d98]

mdelapenya committed:
LPS-XXX Support executing tests in PostgreSQL too

mdelapenya committed:
LPS-XXX Build portal-test-ext properties depending on the DB

mdelapenya committed:
LPS-XXX Use UTF8 on Postgre DB

mdelapenya committed:
LPS-XXX Seems that travis does not read ANT_OPTS there

mdelapenya committed:
LPS-XXX Write properties on each DB value

mdelapenya committed:
LPS-XXX Cat portal-test-ext file (remove)

mdelapenya committed:
LPS-XXX Fix postgres

mdelapenya committed:
LPS-XXX Increase memory

mdelapenya committed:
LPS-XXX Avoid waiting for jpda debug

mdelapenya committed:
LPS-XXX Use ruby compiler
